bip-update-process.md

```diff
@@ -0,0 +1,645 @@
+This may or may not be the original authors of the BIP. The format of each authors header value must be
+
+    Random J. User <address@dom.ain>
+
+If there are multiple authors, each should be on a separate line following [RFC
+2822](https://datatracker.ietf.org/doc/html/rfc2822.html) continuation line conventions.
+
+__Status__: The Status field can take the four values Preliminary, Ready, Active, and Abandoned. The
+[Workflow](#workflow) section below describes the statuses in detail.
+
+__Type__: The Type header specifies the type of BIP: Specification, Informational, or Process.
+
+__Created__: The Created header records the date that the BIP was assigned a number.
+
+__License/License-Code__: See the [BIP Licensing](#bip-licensing) section below for a description of the
+acceptable Licenses and their SPDX License Identifiers.
+
+__Discussion__: The Discussion header is used to point the audience to relevant discussions of the BIP, e.g. the mailing
+list thread in which the idea for the BIP was discussed, a thread where a new version of the BIP was presented, or
+relevant discussion threads on other platforms. Entries take the format "yyyy-mm-dd: link", e.g.
+`2009-09-09: https://www.mail-archive.com/cryptography@metzdowd.com/msg10142.html`.
+
+__Revision__: The Revision header indicates the version of this BIP after a Change Log section has been added.
+
+__Requires__: BIPs may have a Requires header to indicate existing BIPs the new proposal depends on.
+
+__Replaces/Superseded-By__: BIPs may have a Superseded-By header indicating that a BIP has been rendered obsolete by a
+later document; the value is the number of the BIP that replaces the current document. The newer BIP must have a
+Replaces header containing the number of the BIP that it rendered obsolete.
+
+#### Auxiliary Files
+
+BIPs may include auxiliary files such as diagrams. Auxiliary files must be included in a subdirectory for that BIP named
+`bip-XXXX`, where "XXXX" is the BIP number zero-padded to four digits. File names in the subdirectory do not need to adhere to a specific
+convention.
+
+### BIP types
+
+* A **Specification BIP** defines a set of technical rules affecting the interoperability of implementations. The
+  distinguishing feature of a Specification BIP is that it can be implemented, and implementations can be compliant with
+  it. Specification BIPs should come with or refer to a reference implementation.
+* An **Informational BIP** describes a Bitcoin design issue, provides general guidelines, or other information to the
+  Bitcoin community.
+* A **Process BIP** describes a process surrounding Bitcoin, or proposes a change to (or an event in) a process. Process
+  BIPs are like Specification BIPs, but apply to areas other than the Bitcoin protocol itself. They may propose an
+  implementation, but not to Bitcoin's codebase; they often require community consensus; unlike Informational BIPs, they
```

Comment:
Good point, this will need to be amended. I have a bit of a Chesterton’s fence issue here: I am not aware of applications for Process BIPs other than to regulating interactions with the BIP repository or directing protocol development. I’m wondering whether this was intended to refer to software that would inform protocol development decisions like?

Leaving this open to get back to.

Reply:
I changed this section by dropping the part about proposing an implementation:

```diff
 * A **Process BIP** describes a process surrounding bitcoin, or proposes a change to (or an event in) a process. Process
-  BIPs are like Specification BIPs, but apply to areas other than the bitcoin protocol itself. They may propose an
-  implementation, but not to bitcoin's codebase; they often require community consensus; unlike Informational BIPs, they
-  are more than recommendations, and users are typically not free to ignore them. Examples include procedures,
-  guidelines, and changes to the decision-making process.
+  BIPs are like Specification BIPs, but apply to areas other than the bitcoin protocol and bitcoin implementations. They
+  often require community consensus; unlike Informational BIPs, they are more than recommendations, and users are
+  typically not free to ignore them. Examples include procedures, guidelines, and changes to the decision-making
+  process.
```